manim/stream_starter.py

```diff
@@ -0,0 +1,59 @@
+import code
```

Comment:
This, together with the following suggestion, is a quality of life improvement ensuring that the prompt works a bit better (including tab completion and accessing the history using the arrow keys up+down).

```suggestion
import code
import readline
import rlcompleter
```

Reply:
Readline isnt included in stdlib for windows :(( add `pyreadline` as a dependency, if you want to use it.

Reply:
Ah, yes -- I didn't even check that. We can resolve dependencies in the next step; thanks for the pointer!

Reply:
Should this have been made a draft? I'm getting low dog ears right now.

Reply:
I don't think it has to be; don't interpret too much into "next step". I meant it more along the lines of "in one of the next commits after adding these imports". :-)

Reply:
@naveen521kk can we simply add `pyreadline` as a dependency by running `poetry add pyreadline && poetry lock`? Or is there something that we have to consider, given that its only required for Windows?

Reply:
Now that I'm using pyreadline, and I'm 95% sure I've implemented everything, how do I make that last error about `no module named pyreadline` in the tests work.

> Readline isnt included in stdlib for windows :(( add pyreadline as a dependency, if you want to use it.

Cause I really didn't understand that last part and it still worked for me

Reply:
> Cause I really didn't understand that last part and it still worked for me

Add this `pyreadline = { version = "*", markers = "sys_platform == 'win32'" }` to line number 40 of `pyproject.toml` and then run `poetry lock` keeping manim as root directory. That will make this error not occur.

Reply:
I've done something mad to the laptop so I need to give it a half hour of time-out.

This will correct the tests being done by GitHub too, correct?